Source/Behaviors/SnapToPointLayoutBehavior.swift

```diff
@@ -95,40 +95,58 @@ public class SnapToPointLayoutBehavior: BrickLayoutBehavior {
         resetCollectionViewContentInset(collectionViewLayout)
     }
 
+    internal func getFilteredAttributes(layout collectionViewLayout: UICollectionViewLayout, frame: CGRect) -> [BrickLayoutAttributes] {
+        guard let attributes = collectionViewLayout.layoutAttributesForElementsInRect(frame) as? [BrickLayoutAttributes] else {
+            return []
+        }
+
+        let filteredAttributes = attributes.filter {
+            return !self.sectionsToIgnore.contains($0.indexPath.section)
+        }
+
+        guard !filteredAttributes.isEmpty else {
+            return []
+        }
+
+        return filteredAttributes
+    }
+
     func resetCollectionViewContentInset(collectionViewLayout: UICollectionViewLayout) {
         guard let frame = collectionViewLayout.collectionView?.frame else {
             return
         }
 
-        guard let attributes = collectionViewLayout.layoutAttributesForElementsInRect(frame) as? [BrickLayoutAttributes] where !attributes.isEmpty else {
+        let filteredAttributes = getFilteredAttributes(layout: collectionViewLayout, frame: frame)
+
+        guard !filteredAttributes.isEmpty else {
             return
         }
 
         originalTopContentInset = collectionViewLayout.collectionView!.contentInset.top
 
         switch scrollDirection {
         case .Horizontal(let scrollLocation):
-            let firstAttributes = attributes.minElement {
+            let firstAttribute = filteredAttributes.minElement {
```

Comment:
No need to rename `firstAttributes` to `firstAttribute`. The object contains `BrickLayoutAttributes` (with a S)